src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -335,9 +336,16 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
           values: JSON.stringify(queryInstance.values),
           __prismaRawParamaters__: true,
         }
-      } else {
+      } else if ("string" === typeof stringOrTemplateStringsArray) {
         // Called with prisma.raw(string)
         query = stringOrTemplateStringsArray
```

Comment:
Here I would also add the parameters since someone could write `.raw(string with $1, values)`. This is unrelated to this PR though

Reply:
Will do